.github/workflows/post-release.yml

```diff
@@ -14,8 +14,8 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          # stable: 'false' # Keep this line to be able to use rc and beta version of Go (ex: 1.18.0-rc1).
-          go-version: 1.18
+          # stable: 'false' # Keep this line to be able to use rc and beta version of Go (ex: 1.18.0-rc.1).
```

Comment:
Sorry, I'm trying to help but don't have much context, why the "." Is added? I see the former format in the go repo tags. Furthermore, should we take the opportunity to update the comment to 1.19 or do we have to keep it in 1.18 for some reason?

Reply:
It's related to the [setup-go](https://github.com/actions/setup-go)

https://github.com/actions/setup-go#supported-version-syntax

The message is just for the pattern.

Reply:
I improved the comments.